libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationPerformanceEvent.java

```diff
@@ -19,20 +21,36 @@
   private static final SimpleDateFormat dateFormat = new SimpleDateFormat(DATE_AND_TIME_PATTERN, Locale.US);
   private static final String PERFORMANCE_TRACE = "mobile.performance_trace";
   private static final String EVENT_NAME = "event_name";
+  private static final String SCREEN_DENSITY = "screen_density";
 
   private final String event;
   private final String created;
   private final String sessionId;
   private final List<Counter> counters;
   private final List<Attribute> attributes;
+  NavigationPerformanceMetadata metadata;
 
-  NavigationPerformanceEvent(String sessionId, String eventName) {
+  NavigationPerformanceEvent(String sessionId, String eventName,
```

Comment:
> There are now two constructors for the NavigationPerformanceEvent, one for creating an event with metadata, and one for creating an event without metadata. 

Is this necessary? Wondering if we always want to capture the metadata 🤔 I guess so, right?